site/content/how-to/traffic-management/backend-tls-termination.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "Backend TLS Termination"
+description: "Learn how to encrypt HTTP traffic between NGINX Gateway Fabric and your backend pods."
+weight: 600
+toc: true
+docs: "DOCS-000"
+---
+
+In this guide, we will show how to specify the TLS configuration of the connection from the Gateway to a backend pod/s via the Service API object using a [BackendTLSPolicy](https://gateway-api.sigs.k8s.io/api-types/backendtlspolicy/).
```

Comment:
I know this is just a guide, and maybe we'll have more fleshed out docs for describing *why* features are desired, but could we add an additional sentence just describing why someone would want to do this?